lib/engine/step/g_18_al/token.rb

```diff
@@ -10,7 +10,7 @@ def place_token(entity, city, token, teleport: false)
           super
 
           entity.abilities(:assign_hexes) do |ability|
-            next unless ability.name == 'Historical objective'
+            next unless ability.name.start_with == 'Historical objective'
```

Comment:
this is not a method?

it should be name.start_with?('Historical objective')

Reply:
I removed the name/description from the configuration and check for hexes size instead. The other asssign_hexes ability in 18AL has more than 1 hexes to choose from.

Reply:
Yeah, was a bug. But I changed this check to checking for hexes.length instead, so I could remove name/description from the config file.